packages/loader/src/resource-deserialize/resources/scene/SceneParser.ts

```
@@ -66,21 +72,27 @@ export class SceneParser extends HierarchyParser<Scene, ParserContext<IScene, Sc
-        const components = entity.components;
-        for (let j = 0, m = components.length; j < m; j++) {
-          const component = components[j];
-          this._searchDependentAssets(component.methods);
-          this._searchDependentAssets(component.props);
+        const componentIndices = entity.components;
+        if (!componentIndices) continue;
+        for (let j = 0, m = componentIndices.length; j < m; j++) {
+          const comp = components[componentIndices[j]];
+          if (comp.script) {
+            // @ts-ignore
+            context._addDependentAsset(comp.script.$ref, context.resourceManager.getResourceByRef(assetRefToEngine(comp.script)));
```

ESLint

[error] Replace `comp.script.$ref,·context.resourceManager.getResourceByRef(assetRefToEngine(comp.script))` with `⏎··············comp.script.$ref,⏎··············context.resourceManager.getResourceByRef(assetRefToEngine(comp.script))⏎············`

(prettier/prettier)